modules/ROOT/pages/external-identity-index.adoc

```diff
@@ -3,11 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-**** xref:managing-users.adoc[To Configure SAML]
-***** xref:verify-saml-info-task.adoc[To Verify SAML Information]
-***** xref:troubleshoot-saml-assertions-task.adoc[To Troubleshoot SAML Assertions]
-***** xref:update-user-profile-task.adoc[To Update Your User Profile]
-***** xref:single-log-out-task.adoc[To Configure Single Log Out]
+**** xref:managing-users.adoc[Configure SAML for SSO]
+***** xref:verify-saml-info-task.adoc[Verify SAML Information]
+***** xref:troubleshoot-saml-assertions-task.adoc[Troubleshoot SAML Assertions]
+***** xref:update-user-profile-task.adoc[Update Your User Profile]
```

Comment:
`***** xref:update-user-profile-task.adoc[Update Your User Profile]` should be under the `conf-openid-connect-task` section.
Also, the ordering of this block is not the same as in `modules/ROOT/nav.adoc` but I think it should be.